raster/r.latlong/tests/test_r_latlong.py

```diff
@@ -0,0 +1,99 @@
+import os
+import pytest
+import grass.script as gs
+
+
+@pytest.fixture
+def setup_maps(tmp_path):
+    """Set up a GRASS session and create test raster maps."""
+
+    project = tmp_path / "r_latlong_project"
+    gs.create_project(project)
```

Comment:
This creates an XY project (no coordinate reference system), I think you can remove that line.